packit.spec

```diff
@@ -94,6 +94,9 @@ cp files/bash-completion/packit %{buildroot}%{_datadir}/bash-completion/completi
 %{python3_sitelib}/*
 
 %changelog
+* Thu May 05 2022 Matej Focko <mfocko@redhat.com> - 0.50.0-1
+- New upstream release 0.50.0
```

Comment:
gotta break the habit, i noticed that releng uses capitalized messages for rebuilds ;)